sdk/appconfiguration/azure-data-appconfiguration/src/samples/java/com/azure/data/appconfiguration/ConfigurationAsyncClientJavaDocCodeSnippets.java

```diff
@@ -16,6 +16,7 @@
  */
 public class ConfigurationAsyncClientJavaDocCodeSnippets {
 
+    private static final String NO_LABEL = null;
```

Comment:
Doesn't this exist in the `ConfigurationSetting` class? If the one in `ConfigurationSetting` isn't public lets  make it so.